pkg/kubelet/kubelet_lxcfs.go

```diff
@@ -0,0 +1,60 @@
+package kubelet
+
+import (
+	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+	"k8s.io/kubernetes/pkg/kubelet/container"
+)
+
+//LxcfsMounts is
+func LxcfsMounts() []container.Mount {
+	return []container.Mount{
+		{
```

Comment:
Can we have a simple way to organize this struct member?
e.g. 

```go
const LXCFS_PROC_PATH = "/var/lib/lxcfs/proc/"

var lxcfsProcEntry = []string {"cpu","meminfo","diskstats","uptime","stat"}

func LxcfsMounts() []container.Mount {
	var mounts [] container.Mount
	for _, entry := range lxcfsProcEntry {
		mount := container.Mount{
			Name:           fmt.Sprintf("lxcfs-%s",entry),
			ContainerPath:  fmt.Sprintf("/proc/%s",entry),
			HostPath:       LXCFS_PROC_PATH + entry,
			ReadOnly:       true,
			SELinuxRelabel: false,
			Propagation:    runtimeapi.MountPropagation_PROPAGATION_HOST_TO_CONTAINER,
		}
		mounts = append(mounts,mount)
	}
	return mounts
}
```

For this, another concern is that once lxcfs support another `/proc file api`，we can just simply change the `lxcfsProcEntry`.

\cc @resouer  what do you think about this

^_^·

Reply:
OK I will optimize and test it.